src/managers/input/Swipe.cpp

```diff
@@ -225,6 +227,14 @@ void CInputManager::onSwipeUpdate(wlr_pointer_swipe_update_event* e) {
         return;
     }
 
+    if (*PSWIPEDIRLOCK) {
+        if (m_sActiveSwipe.initialDirection != 0 && m_sActiveSwipe.initialDirection != (m_sActiveSwipe.delta < 0 ? -1 : 1)) {
```

Comment:
I'd rather have a minimum threshold for the thing to initiate the lock. When you first put your fingers, without moving them, it might jump a few pixels around left and right